apps/prairielearn/src/models/course.ts

```diff
@@ -12,3 +15,55 @@ export async function selectCourseById(course_id: string): Promise<Course> {
     CourseSchema,
   );
 }
+
+export function getLockNameForCoursePath(coursePath: string): string {
+  return `coursedir:${coursePath}`;
+}
+
+export async function getCommitHash(coursePath: string): Promise<string> {
```

Comment:
We want functions to name what they're operating on:

```suggestion
export async function getCourseCommitHash(coursePath: string): Promise<string> {
```

Reply:
Done in a94edb6c8.